flight-website/app/templates/application.hbs

```diff
@@ -1,5 +1,14 @@
 {{page-title "Flight icons"}}
 <DsHeader />
+<div class="ds-beta-banner">
+  <strong>Notice</strong>: there is a new "beta" version of this website:
```

Comment:
I would normally make a colon following bolded text bold also. This seems to be a common pattern but not sure if we have an accepted style rule we follow.

Reply:
Fixed.